src/components/views/dialogs/TextInputDialog.js

```diff
@@ -125,7 +129,8 @@ export default class TextInputDialog extends React.Component {
                     </div>
                 </form>
                 <DialogButtons
-                    primaryButton={this.props.button}
+                    primaryButton={this.state.loading ? _t("Loading...") : this.props.button}
```

Comment:
Could we have `Loading...` come in as a prop which the caller specifies to be more generic?

Reply:
Yes we can have that as an optional prop